cdap-app-templates/cdap-etl/cdap-data-pipeline-base/src/main/java/io/cdap/cdap/datapipeline/service/ConnectionHandler.java

```diff
@@ -155,6 +165,8 @@ public void getConnection(HttpServiceRequest request, HttpServiceResponder respo
                             "Getting connection in system namespace is currently not supported");
         return;
       }
+      contextAccessEnforcer.enforce(new SystemAppEntityId(namespace, "pipeline", "connection",
```

Comment:
Here it is good to create a `ConnectionEntityId` which extends `SystemAppEntityId` to create these entities in an easy way, i.e,  `new ConnectionEntityId(connection-id)`

Reply:
This applies to all places which uses `SystemAppEntityId`

Reply:
ack